matchtree.go

```diff
@@ -963,7 +969,11 @@ func (d *indexData) newMatchTree(q query.Q) (matchTree, error) {
 		}, nil
 
 	case *query.Symbol:
-		subMT, err := d.newMatchTree(s.Expr)
+		// Disable WordMatchTree since we don't support it in symbols yet.
+		optCopy := opt
```

Comment:
Why do we need to copy here? Doesn't NewMatchTree already get a shallow copy of opt?

Reply:
feels safer to avoid mutating, but yes in this case we can avoid copying. Gonna keep it this way since it feels safer to future mutations.